pages/api/login.ts

```diff
@@ -1,29 +1,49 @@
 import type { NextApiRequest, NextApiResponse } from 'next'
 import { PrismaClient } from '@prisma/client'
+import bcrypt from 'bcrypt'
+import jwt from 'jsonwebtoken'
 
 const prisma = new PrismaClient()
 
 type Data = {
   message: string,
-  xBhaktToken: any
+  xBhaktToken?: any
 }
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse<Data>
 ) {
   if (req.method === 'POST') {
-    const password_hash = req.body.password.trim();
+    const password = req.body.password.trim();
     const email = req.body.email.trim().toLowerCase();
 
-    prisma.bhakt.findFirstOrThrow({ where: { email: email, password_hash: password_hash } })
-      .then(data => res.status(200).json({
-        message: "Successfully loggedin",
-        xBhaktToken: data.id
-      }))
+    prisma.bhakt.findFirstOrThrow({ where: { email: email } })
+      .then(data =>
+        bcrypt.compare(password, data.password_hash).then((match) => {
+          if (!match) {
+            res.status(400).json({
+              message: "Wrong Credential!",
+            })
+          }
```

Comment:
We have to stop execution of our code here using return or keep the rest of the code in else part